terraso_backend/apps/graphql/schema/sites.py

```diff
@@ -35,6 +36,19 @@ class Meta:
         connection_class = TerrasoConnection
 
 
+class SiteFilter(django_filters.FilterSet):
+    class Meta:
+        model = Site
+        fields = ["name", "owner", "project", "project__id"]
+
+    order_by = django_filters.OrderingFilter(
+        fields=(
+            ("updated_at", "updated_at"),
+            ("created_at", "created_at"),
+        )
+    )
```

Comment:
question: since we need to store sites for offline, does it make sense to bother ordering them in the backend since we will need to implement ordering in the frontend anyway?

Reply:
I would suspect that for pagination and stuff this will still be useful.